src/tests/issues.targets

```diff
@@ -1157,6 +1166,11 @@
         <ExcludeList Include="$(XunitTestBinBase)/Loader\classloader\MethodImpl\CovariantReturns\UnitTest\UnitTest_GVM\*" />
         <ExcludeList Include="$(XunitTestBinBase)/Loader\classloader\MethodImpl\CovariantReturns\UnitTest\UnitTestDelegates\*" />
         <ExcludeList Include="$(XunitTestBinBase)/Loader\classloader\MethodImpl\CovariantReturns\UnitTest\OverrideMoreDerivedReturn\*" />
+
+        <!-- Requires Debug build -->
+        <ExcludeList Include="$(XunitTestBinBase)/JIT\Directed\debugging\poisoning\poison\poison\*" />
+        <ExcludeList Include="$(XunitTestBinBase)/JIT\Methodical\eh\finallyexec\nestedfinallycall_d\*" /> <!-- probably RyuJIT bug -->
```

Comment:
> probably RyuJIT bug

File one?

Reply:
I honestly don't understand the point of these JIT tests where they say "I never want to run this with optimizations". To me it means "I don't care about bugs in this with optimizations on".

Reply:
The test is built two different ways, once with optimization disabled and once with it enabled. Works fine both ways with the normal runtime. 

If you add repro instructions to the bug you filed, I will take a look at why it fails with NAOT.